imports/plugins/core/accounts/server/no-meteor/mutations/inviteShopMember.js

```diff
@@ -0,0 +1,140 @@
+import _ from "lodash";
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import ReactionError from "@reactioncommerce/reaction-error";
+import getCurrentUserName from "/imports/plugins/core/accounts/server/no-meteor/util/getCurrentUserName";
+import getDataForEmail from "/imports/plugins/core/accounts/server/util/getDataForEmail";
```

Comment:
Since it no longer has Meteor deps, you can move `getDataForEmail.js` from `server/util` to `server/no-meteor/util`.